src/plugins/timeago.js

```diff
@@ -9,7 +9,8 @@ export default ({
         name: "Timeago",
         locale: "en",
         locales: {
-            ru: require("date-fns/locale/ru")
+        ru: require("date-fns/locale/ru"),
+	fr: require("date-fns/locale/fr"),
```

Comment:
```suggestion
	fr: require("date-fns/locale/fr")
```
Indentation needs to be fixed as well